_posts/2019-05-15-cockpit-194.md

```diff
@@ -0,0 +1,53 @@
+---
+title: Cockpit 194
+author: lis
+date: 2019-05-15
+tags: cockpit linux
+slug: cockpit-194
+category: release
+summary: Firewall page improvements and a new switch
+comments: 'true'
+---
+
+Cockpit is the [modern Linux admin interface](https://cockpit-project.org/). We
+release regularly. Here are the release notes from version 194.
+
+### Firewall: Add services to a specific zone
+
+Services can now be added to and removed from specific zones.
+
+![Adding services to zones](/images/firewall-add-services-to-zones.png)
+
+Furthermore, the firewall page will now give limited feedback in case of
+errors.
+
+![Firewall error message](/images/firewall-error-message.png)
+
+### Redesigned on/off switch
+
+This release of Cockpit features a redesigned on/off switch.  Compared
+with the old version, the new switch is easier to use (with a bigger
+clickable area) and uses iconography to avoid translation problems.  The
+new design is also a better match for PatternFly 4.
```

Comment:
This is not that interesting for users, but maybe for developers. How about linking to https://pf4.patternfly.org/components/Switch/examples/ ?